### DISCUSSION TITLE
Is Philips Iris livingcolor 7099960PH supported?

### DISCUSSION THREAD

**Discussion by Novocaine85** [Koenkk/zigbee2mqtt#9424] [Devices]
Hi,

this page say that philips iris livingcolor is supported https://www.zigbee2mqtt.io/devices/7099930PH.html

however, my model is 7099960PH. the number difference should be just the lamp color (mine is white, the 7099930 should be black.)

However after many tries i'm unable to pair the device. I can see the packets through cc2591 sniffer. I can see the lights of my coordinator blinking while in permit join, but no logs and the device can't join.

So, is it really supported? since the product code is different, does it need to be duplicate inside zigbee2mqtt or something like that in order do make it work?

thanks

**Comment by gvlekke** [comment]
did you already try to reset it with the hue dimmer remote? that helped with me with pairing. altho it stills says not supported and I also dont see a model number when i generate the external definition
